<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RPSGame</class>
 <widget class="QMainWindow" name="RPSGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPSGame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="rockButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Rock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paperButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Paper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scissorsButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Scissors</string>
    </property>
   </widget>
   <widget class="QLabel" name="computerChoiceLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Computer chose ....</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Result Awaited ....</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="userScoreLCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="computerScoreLCD">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userScoreLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>5</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>     USER SCORE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="computerScoreLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPUTER SCORE:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>RPSGame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
